mltsp/util.py

```diff
@@ -0,0 +1,25 @@
+import subprocess
```

Comment:
This already exists as ``docker_util``, so we should merge those two submodules.